encrypted-media-respec.html

```diff
@@ -2310,15 +2310,15 @@
         </section>
         
         <section id="media-key-session-destroyed">
-          <h4>MediaKeySession destroyed</h4>
+          <h4>MediaKeySession Destroyed</h4>
           <p>
-            This algorithm performs steps that are necessary when a <a>MediaKeySession</a> is destroyed without
-            the <a def-id="session-closed-algorithm"></a> algorithm having been run.
+            This algorithm performs steps that are necessary when a <a>MediaKeySession</a> is destroyed and
+            the <a def-id="session-closed-algorithm"></a> algorithm has not been run.
           </p>
-          <p>The following steps are run:</p>
+          <p>The following steps are run in parallel [to the main thread/event loop]:</p>
```

Comment:
Thanks. I meant that the bracketed text was optional, not to actually be included bracketed. If it is included, pick " thread" or "event loop". It looks like we use "event loop" in multiple places. Thus:
>s/[to the main thread/event loop]/to the main event loop/